src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
 
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Comments } from './types/Comments';
+import { Posts } from './types/Posts';
+
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+function getUsersById(userId:number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getPostsById(postId:number):Comments[] {
+  return commentsFromServer.filter(comment => comment.postId === postId) || [];
+}
+
+const postsArray: Posts[] = postsFromServer.map((post) => {
+  return (
+    {
+      ...post,
+      user: getUsersById(post.userId),
+      comments: getPostsById(post.id),
+    }
+  );
+});
```

Comment:
variable in plural means array
```suggestion
const posts: Posts[] = postsFromServer.map((post) => ({
      ...post,
      user: getUsersById(post.userId),
      comments: getPostsById(post.id),
    }));
```

Reply:
Not fixed 😢 
![image](https://user-images.githubusercontent.com/99403198/230348261-06e2d4ff-87a8-45ad-b39a-6fcc1e0c51bc.png)